ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -82,7 +88,10 @@ public boolean onTouchEvent(MotionEvent ev) {
       ReactScrollViewHelper.emitScrollEndDragEvent(this);
       mDragging = false;
     }
-    return super.onTouchEvent(ev);
+    if (mScrollEnabled) {
```

Comment:
you can also do this on one line
`return mScrollEnabled && super.onTouchEvent(ev);`